packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -94,6 +110,18 @@ export interface MarketInfoOutcome {
   volume: string;
 }
 
+export enum MarketInfoReportingState {
```

Comment:
Did these move? They are somewhere else, I thought. The UI references ‘Getters.Markets.MarketReportingState.PreReporting’

Reply:
Oh good catch. I'll rename it.